config/secrets.yml

```diff
@@ -19,9 +19,13 @@
 
 development:
   secret_key_base: d22cd95910c4425ab8adfd2aa968b845775c6ffff23f9cdb337bc6f210199ca97736ed261d06bf69242f6483a8e6c474ab88f02dabb9865f9911ff215da31695
-
+  slack_token: "xoxp-235986621104-238493557106-238649524195-b886dacfa83620d7c79b483114198cf9"
```

Comment:
Is this a real token? It might be better as an environment variable like SECRET_KEY_BASE below. That would make sense for all the environments. Someone with Heroku access to the app would need to set this up before deploying.

The secret should be rolled as well.